packages/icons/dist/webfont.css

```diff
@@ -0,0 +1,48 @@
+@font-face {
+    font-family: webfont;
+    src: url("./webfont.eot");
+    src: url("./webfont.eot?#iefix") format("embedded-opentype"), url("./webfont.woff2") format("woff2"), url("./webfont.woff") format("woff"), url("./webfont.ttf") format("truetype"), url("./webfont.svg#webfont") format("svg");
+    font-style: normal;
+    font-weight: 400;
+}
+
+.webfont {
```

Comment:
webfont?

Reply:
well, someone changed the `.gitignore` :P

Reply:
xD